plone/synchronize/tests.py

```diff
@@ -46,14 +46,14 @@ def test_instance_method(self):
         except IndexError:
             raised = True
 
-        self.failUnless(raised)
+        self.assertTrue(raised)
```

Comment:
I like `self.assertRaises(IndexError, shared_stack.pop)` better, as it is clearer to me, but your solution totally works and is less code changed.

Reply:
Ah, I did not look carefully enough at your patch and just did a quick search-and-replace myself. I will do an extra commit with assertRaises now (used as context manager).